### ISSUE TITLE
Two transactions “transaction not found” error

### ISSUE THREAD

**Issue by isajj** [bitpay/copay#7398]
I sent two small amounts of bitcoin from my Copay wallet to a new Segwit address on a Ledger Nano S. I set the fee at 10 sat/byte (understanding it would be pending for some time). Both transactions are pending on Copay, but unlike previous pending transactions, they’re showing a “transaction not found” error when I try to see them on the blockchain. I’ve tried to manually broadcast the transaction on multiple sites and all are giving an error.

The transaction hashes are as follows:

bc7ce307e399d120953626eab4e3436cff0889930d546c5d55aecda4f668a5d0

6250f0eeb97fca00e5eb03c27c43577a1b512247292c3a8e55ac3e0b896b75a4

Is the error happening because the fee is too low? Or is it because I’m sending to a Segwit address? Or something else? And when will I be able to access my coins again? Anything you might be able to advise would be greatly appreciated.

**Comment by blueknight02** [comment]
I am also having this issue. I would like to know if it is because of the high network traffic, or a more serious issue with Copay. 

I've now tried sending 2 transactions(both with 11 sat/byte) that haven't shown up on the network.

**Comment by invertedNormal81** [comment]
I am also having the same issue. A friend and I tried to empty our CoPay multisig wallet today.  The Super Economy fee recommendation was 160$ to send 250$ worth of bitcoin.  So we figured out how to set a Custom fee and we set it to 0.00065952 BTC... we thought that was fair and worth the risk.  The payment appeared to send, but it did not.  The wallet said it was broadcast, but it was not.  The sending and receiving addresses show no mempool Tx pending confirmation (sending:3EhnNYSgWy3pbbFjzr1cucHC9zQ9CGePBf ; receiving:1HWagbpikhRwP2DxCK77uBkaKZp5yCxR3X)

The supposed TxID via transactions export from the desktop CoPay wallet (This TxID does not resolve in Insight): 7008414b802f0223769f712045af3f033d60466b21f0687088938082cee5b390

In the CoPay wallet the sending transaction has said sent "a few seconds ago" for over a day now.
![screenshot](https://user-images.githubusercontent.com/34420941/33806996-dbb02ac4-dd95-11e7-9b19-0575d3734970.png)

Here is the wallets entire transaction history

Please help address this.  Thank you

2017-12-10T15:58:09.000Z | 1HWagbpikhRwP2DxCK77uBkaKZp5yCxR3X |   | -0.01585905 | BTC | 7008414b802f0223769f712045af3f033d60466b21f0687088938082cee5b390 | Brian | Brian:accept - James:accept -
-- | -- | -- | -- | -- | -- | -- | --
2017-12-10T15:58:09.000Z | Bitcoin Network Fees |   | -0.00065952 | BTC |   |   |  
2017-12-04T17:50:13.000Z |   |   | 0.00208037 | BTC | 6843224e131700d0a13b9ad2ff7153a1372c23c016dd0e178c44d7db58d1cf36 |   |  
2017-11-14T23:37:56.000Z |   |   | 0.00159316 | BTC | 6c77fac2b76e074ff2eaca89ed5b227d7291b1ad27719cddb13187c08ed34eba |   |  
2017-10-29T03:41:30.000Z |   |   | 0.00158278 | BTC | 9a2533f00742ee6cb11233b20e3f1dced6bf8851b5e154247ac757cadc2791f5 |   |  
2017-10-14T02:47:09.000Z |   |   | 0.00155855 | BTC | d421439002e92e6f714f46468e76cfa1a0bf30911c31b9a50d6693f23752d01e |   |  
2017-09-16T18:56:34.000Z |   |   | 0.0015138 | BTC | 1bcb4242f01a51f539761fca92f8120b647d42bf380f120375085e0217f68ff5 |   |  
2017-09-05T23:06:00.000Z |   |   | 0.00154267 | BTC | 59f82afa21e241adab2f9bb6c8d82b460a16f9a6e4d9735e36e493d6459fcdfb |   |  
2017-08-27T03:51:09.000Z |   |   | 0.00156037 | BTC | 1b6796a861e309ade2c4e263fd40160cecdfe4e69e05d488c16655630244d258 |   |  
2017-08-12T13:05:27.000Z |   |   | 0.00152028 | BTC | 2eb1411b589a142a115f7edc17cdeaceb7f249e1ad8ac00ef3ba25276558bd9f |   |  
2017-07-31T12:39:29.000Z |   |   | 0.0020269 | BTC | 06bdd1d19a7478c575f716198b7f3a36bb01184f7e485e8073b911d0310c74db |   |

**Comment by isajj** [comment]
Hi all, managed to fix this (not easily) so thought I’d post an update. The transactions remained stuck in Copay, from what I gather it’s because of  the size of the mempool at the moment being so backlogged that servers are just rejecting lower fee txs out of hand. Anyway, use the following at your own risk (as it involves decrypting your private key) but managed to follow the instructions at the following link to import the wallet to electrum and resend the amounts using a higher but still relatively low fee (~100sat/byte).

https://github.com/bitpay/copay/issues/5859

In short, use https://bitwiseshiftleft.github.io/sjcl/demo/ to derive the xprivkey, and use https://bip32jp.github.io/english/ to derive the account extended private key, which you can then enter into Electrum. If you do it right you should see your full balance available which you can then empty into your destination wallet - sent within half an hour for me.

**Comment by invertedNormal81** [comment]
Thanks isajj,

I was actually able to recover from a multisig wallet using BitPay's own recovery tool.  All I needed was the Backup Phrases for both signers.  It allowed me to send the funds to an address with a .001 BTC fee instead of the 160$ BTC fee the app wanted to charge.

Repo: https://github.com/bitpay/copay-recovery
Tool: https://bitpay.github.io/copay-recovery/

**Comment by omietrice** [comment]
Hi Isajj, thank you for the update in hopes I can fix this. I am having a similar issue. It says the transaction has been broadcasted but is not appearing in the mempool. My balance is constantly fluctuating. It's not being rejected back my wallet though. Will the method you used work if it's not sent to a segwit address as well?

This is very worrysom, I've been using copay over a year and am starting to be concerned.

**Comment by JDonadio** [comment]
You can open the view, copy the url (before the insight page did load completely) and use the transaction id in any other block explorer to check their status. Sorry about that, we must fix our public insight to show the transactions properly.
